Common/Orders/OrderEvent.cs

```diff
@@ -33,11 +33,21 @@ public class OrderEvent
         public Symbol Symbol;
 
         /// <summary>
+        /// The date and time of this event (UTC).
+        /// </summary>
+        public DateTime Time;
+
+        /// <summary>
         /// Status message of the order.
         /// </summary>
         public OrderStatus Status;
 
         /// <summary>
+        /// The fee associated with the order (always positive value).
+        /// </summary>
+        public decimal OrderFee;
```

Comment:
Also, we need to verify this is being set by everyone who creates this. Recommend making it a required constructor parameter. I'm mostly thinking of the brokerage implementation who don't know to set this value. If the call site doesn't have the required info, they can pass zero, but at least it forces the question to all who instantiate this type.